test/E2E/HttpTrigger.Tests.ps1

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+Describe 'HttpTrigger Tests' {
+    BeforeAll {
+        $FUNCTIONS_BASE_URL = 'http://localhost:7071'
+        & "$PSScriptRoot/setupE2Etests.ps1"
+        { Invoke-RestMethod $FUNCTIONS_BASE_URL } | Should -Not -Throw -Because 'The E2E tests require a Function App to be running on port 7071'
+    }
+
+    AfterAll {
+        Get-Job -Name FuncJob -ErrorAction SilentlyContinue | Stop-Job | Remove-Job
+    }
+
+    It "Test basic HttpTrigger function - Success" -TestCases @(
+        @{ 
+            FunctionName = 'TestBasicHttpTrigger'
+            InputNameData = 'Atlas'
+            ExpectedStatusCode = 200
```

Comment:
Prefer using `[HttpStatusCode]::OK` (along with `using namespace System.Net` at top) making it more readable.  Applies to all the status codes below